src/nwb/ecephys/ElectricalSeries.hpp

```diff
@@ -81,7 +77,7 @@ class ElectricalSeries : public TimeSeries
   /**
    * @brief Path to the electrodes table this time series references
    */
-  std::string electrodesTablePath;
+  std::string electrodesTablePath = "/general/extracellular_ephys/electrodes/";
```

Comment:
```suggestion
  inline static const std::string electrodesTablePath = "/general/extracellular_ephys/electrodes/";
```

Reply:
Actually, I would make `electrodeTablePath` a `static const` member of `ElectrodeTable` such that we can just grab from the `ElectrodeTable` class, rather than making it a property of `ElectricalSeries`. Alternatively, it could also be a `static const` member of `NWBFile` since it really defines where the `ElectrodesTable` goes.

Reply:
I see, yes that looks better and minimizes the path duplication. I implemented on `ElectrodeTable`